addons/tidb/dataprotection/backup.sh

```diff
@@ -25,7 +25,8 @@ function getToolConfigValue() {
 access_key_id=$(getToolConfigValue access_key_id)
 secret_access_key=$(getToolConfigValue secret_access_key)
 endpoint=$(getToolConfigValue endpoint)
-bucket=$(getToolConfigValue root)
+# workaround for "access_key_id = root" 
+bucket=$(getToolConfigValue "root =")
```

Comment:
should update `getToolConfigValue()` to overcome the problem instead of working around it.

for example:

```diff
function getToolConfigValue() {
    local var=$1
-   cat $toolConfig | grep "$var" | awk '{print $NF}'
+   cat $toolConfig | grep "^$var" | awk '{print $NF}'
}
```

Reply:
good point